src/radical/entk/appman/appmanager.py

```diff
@@ -384,14 +386,12 @@ def run(self):
 
                 if (not self._wfp.check_processor()) and (self._cur_attempt <= self._reattempts):
 
-                    """
-                    If WFP dies, both child threads are also cleaned out.
-                    We simply recreate the wfp object with a copy of the workflow
-                    in the appmanager and start the processor.
-                    """
+                    # If WFP dies, both child threads are also cleaned out.
+                    # We simply recreate the wfp object with a copy of the workflow
+                    # in the appmanager and start the processor.
 
                     self._prof.prof('recreating wfp obj', uid=self._uid)
-                    self._wfp = WFProcessor(
+                    self._wfp = WFprocessor(
                         sid=self._sid,
```

Comment:
How has that ever worked?  The same code is in `master`.  I am puzzled...

Reply:
@vivek-bala : I really could not figure this one out - this seems to have been typed like that since, like, forever.  Any idea why python did not complain?

Reply:
I think the code was never reached from any use case, and as a result python's interpreter never tried to run it.

Reply:
Giannis is correct. I think that code part was not reached in most occasions.